src/app/api/auth/[...nextauth]/route.ts

```diff
@@ -55,7 +75,9 @@ export const authOptions: AuthOptions = {
       },
       clientId: AppConstants.OAUTH_CLIENT_ID,
       clientSecret: AppConstants.OAUTH_CLIENT_SECRET,
+      // Parse data returned by FxA's /userinfo/
       profile: async (profile: FxaProfile) => {
+        log.debug("fxa-confirmed-profile-data", profile);
```

Comment:
It's not; it's a port of this line: https://github.com/mozilla/blurts-server/blob/d4cb77862d91a61e0debf1032301428c2019ee04/src/controllers/auth.js#L71

Though the concern still stands. @mansaj added it in 0d89c4b0ab6c9b65e62e235f22989bda39d469f9, so maybe he can clarify whether this can/should be removed?

Reply:
We can remove this @Vinnl @pdehaan. It was just for debugging purpose (should be omitted in prod).

Reply:
Oh right, it doesn't log on prod! Then it looks fine and potentially helpful to me :)